diffkemp/simpll/DebugInfo.h

```diff
@@ -45,6 +48,9 @@ class DebugInfo {
         calculateGEPIndexAlignments();
     };
 
+    /// Maps structure type and index to struct member names
+    ElementIndexToNameMap EINM;
```

Comment:
Don't use such long and unclear abbreviation for variable name. I'd prefer something as StructFieldNames.